packages/kit/src/views/Send/pages/SendDataInput/SendDataInputContainer.tsx

```
@@ -3,15 +3,19 @@
 
 import { useRoute } from '@react-navigation/core';
 import BigNumber from 'bignumber.js';
+import { utils } from 'ethers';
 import { isNaN, isNil } from 'lodash';
```

Biome

[error] Do not shadow the global "isNaN" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)